sources/packages/web/src/assets/css/base.scss

```diff
@@ -417,3 +417,8 @@ div.v-field__field input[type="date"] {
     @extend .font-bold;
   }
 }
+
+.remove-side-padding {
+  padding-left: 0 !important;
+  padding-right: 0 !important;
+}
```

Comment:
Please remove the style. If such a style is required, it must be applied globally to all the buttons.